lib/app.go

```diff
@@ -139,7 +135,7 @@ func appState(app *App, pod *pkgPod.Pod) error {
 	}
 
 	app.State = AppStateCreated
-	createdAt := fi.ModTime().UnixNano()
```

Comment:
I'm confused by this change, as comments in `lib/types` explictly say "nanoseconds since epoch" and CRI seems to agree on [that](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/api/v1alpha1/runtime/api.proto#L601). @yifan-gu do you remember what's going on here?